packages/nitro-protocol/gas-benchmarks/vanillaSetup.ts

```diff
@@ -49,3 +60,25 @@ beforeEach(async () => {
 afterAll(async () => {
   await ganacheServer.close();
 });
+
+expect.extend({
+  async toConsumeGas(
+    received: any, // TransactionResponse
+    benchmark: number
+  ) {
+    const {gasUsed} = await received.wait();
+    const pass = (gasUsed as BigNumber).eq(benchmark); // This could get replaced with a looser check with upper/lower bounds
+    if (pass) {
+      return {
+        message: () => `expected to NOT consume ${benchmark} gas, but did`,
+        pass: true,
+      };
+    } else {
+      return {
+        message: () =>
+          `expected to consume ${benchmark} gas, but actually consumed ${(gasUsed as BigNumber).toNumber()} gas. Consider updating the appropriate number in gas.ts!`,
```

Comment:
```suggestion
    const gasUsed: BigNumber = (await received.wait()).gasUsed as BigNumber;
    const pass = gasUsed.eq(benchmark); // This could get replaced with a looser check with upper/lower bounds
    if (pass) {
      return {
        message: () => `expected to NOT consume ${benchmark} gas, but did`,
        pass: true,
      };
    } else {
      return {
        message: () =>
          `expected to consume ${benchmark} gas, but actually consumed ${gasUsed.toNumber()} gas. Consider updating the appropriate number in gas.ts!`,
```
This might be a little easier to read, with all of the type assertion on one line.